src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs

```diff
@@ -46,7 +46,7 @@ private void InitializeArgs(NativeCompileSettings config)
             argsList.Add($"\"{inputFilePath}\"");
             
             // System.Private.CoreLib Reference
-            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "System.Private.CoreLib.dll");
+            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "*.dll");
```

Comment:
These are like standard libraries and we may add more to the list, and if we refactored code and made more DLLs, we would have to keep updating the CLI.